### PR TITLE
Multiple attributes per node

### PR DESCRIPTION
**Pull request by elijahbenizzy**

### CODE REVIEW

**Comment by elijahbenizzy** [comment]
image

**Comment by elijahbenizzy** [comment]
Didn't add tests to the sdk cause this wasn't explicitly tested before, this is a very small change

File: ui/sdk/src/hamilton_sdk/api/constants.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 HAMILTON_API_URL = "http://localhost:8241"
-HAMILTON_UI_URL = "http://localhost:8242"
+HAMILTON_UI_URL = "http://localhost:8241"  # optimizing for mini-mode
```
Comment (skrawcz) [inline ui/sdk/src/hamilton_sdk/api/constants.py:2]: err -- docs need to be updated then for use with docker then right?

Comment (skrawcz) [reply ui/sdk/src/hamilton_sdk/api/constants.py:2]: or there needs to be a way to know which URL to use that the SDK can figure out

Comment (elijahbenizzy) [reply ui/sdk/src/hamilton_sdk/api/constants.py:2]: See commit notes, this was the quickest change but yeah we can update docs

Comment (zilto) [reply ui/sdk/src/hamilton_sdk/api/constants.py:2]: side note: `127.0.0.1` should be preferred to `localhost`  [ref](https://www.youtube.com/watch?v=98SYTvNw1kw)

Comment (elijahbenizzy) [reply ui/sdk/src/hamilton_sdk/api/constants.py:2]: Hmm I'm generally against using a hardcoded value for the purpose of small optimizations, although there are other reasons... Good video.

**Review by skrawcz — APPROVED** [review]
yes, but you need to add to docs on how to change the URL to point to wherever the UI lives.

**Pull request merged**